### ISSUE TITLE
Applying the sample to document classification

### ISSUE THREAD

**Issue by JulianGerhard21** [globality-corp/sklearn-hierarchical-classification#15]
Hi,

it seems your work is exactly what I was looking for - but I got one question. I've got the following problem:

I'm currently approaching a classification problem with the following situation:

The labels are always 5 digits long, e.g.:

`99923 this is sample document one`
`56743 this is sample document two`
`...`

where the first single digit stands for a main category, every following digit for a subcategory and so on. The subcategories depend on the main category and are NOT equivalent for each maincategory. 

Because my training data is limited (around 80k samples overall), I decided to use only the first digit to estimate the main-category and I got pretty good results with ~93% accuracy without any preprocessing which yet has to be done.

`5 - this is a sample of maincategory 5`
`9 - this is a sample of maincategory 9`
`...`

**My question:** 
Is it possible to adopt your approach to my classification problem in a way that I train a classifier that predicts the whole label and not only the first digit and if so, can you give me a hint how to modify your code to work with a csv with the format: "label \t string" ?
